src/comparator.js

```diff
@@ -10,15 +10,15 @@ var _curry1 = require('./internal/_curry1');
  * @since v0.1.0
  * @category Function
  * @sig (a, b -> Boolean) -> (a, b -> Number)
- * @param {Function} pred A predicate function of arity two.
- * @return {Function} A Function :: a -> b -> Int that returns `-1` if a < b, `1` if b < a, otherwise `0`.
+ * @param {Function} pred A predicate function of arity two. Return `true` if the first argument is less than the second, `false` otherwise
```

Comment:
Perhaps joining these two into

    A predicate function of arity two which will return `true` if the first argument 
    is less than the second, `false` otherwise